packages/wethegit-components/src/components/accordion/accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react"
+
+import styles from "./accordion.module.scss"
+import AccordionItem, { CardsData } from "./component/AccordionItem"
+import cards from "./content.js"
+
+export interface AccordionProps
+  extends Omit<React.HTMLAttributes<HTMLDivElement>, "aria-label"> {
+  "aria-label": string
+  toggle: () => void
+  className?: string
+}
+
+export function Accordion({}: AccordionProps) {
+  const [active, setActive] = useState<number | null>(null)
+  const icon = "chevron" // Define your icon name here
+
+  const handleToggle = (index: number) => {
+    active === index ? setActive(null) : setActive(index)
```

Comment:
Whenever using the current state to determine a new use, please use a function
https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state